value_separation.go

```diff
@@ -150,6 +152,102 @@ func uniqueInputBlobMetadatas(levels []compactionLevel) []*manifest.PhysicalBlob
 	return metadatas
 }
 
+// blobRefValueLivenessState tracks the liveness of values within a blob value
+// block via a sstable.BitmatRunLengthEncoder.
+type blobRefValueLivenessState struct {
```

Comment:
what do you think about moving this onto the `sstable.RawColumnWriter` and moving the integration point to `RawColumnWriter.AddWithBlobHandle`? That would ensure that every blob file has this value liveness state for all the blob handles it encodes, even if it's constructed through some other means (eg, if someday we allow ingesting a sstable and a blob file it references together).